Tests/CoreDataLinkDeletionTests.m

```diff
@@ -50,12 +56,12 @@ -(void)testNoLongerExistingLinksGetDeletedOnSync {
         XCTAssertEqual(0U, [self.persistenceManager fetchAssetsFromDataStore].count, @"");
         XCTAssertEqual(2U, [self.persistenceManager fetchEntriesFromDataStore].count, @"");
 
-        LinkedEntry* entry = [self.persistenceManager fetchEntryWithIdentifier:@"1sPD1WORSoyCEKqyM00uck"];
+        LinkedEntry* entry = (LinkedEntry*)[self.persistenceManager fetchEntryWithIdentifier:@"1sPD1WORSoyCEKqyM00uck"];
```

Comment:
Just out of curriosity: It is not possible to indirectly cast the result anymore? The explicit cast needs to be there?